examples/plugins/src/lighthouse/src/lighthouse.plugin.e2e.test.ts

```diff
@@ -0,0 +1,41 @@
+import { join } from 'node:path';
+import { describe, expect, it } from 'vitest';
+import { executePlugin } from '@code-pushup/core';
+import { LIGHTHOUSE_URL } from '../mock/constants';
+import { PLUGIN_SLUG as slug } from './index';
+import { create } from './lighthouse.plugin';
+
+describe('lighthouse-create-export-execution', () => {
+  it('should return PluginConfig that executes correctly', async () => {
+    const pluginConfig = await create({
+      url: LIGHTHOUSE_URL,
+      outputPath: `${join('tmp', 'lighthouse-report.json')}`,
+    });
+    await expect(executePlugin(pluginConfig)).resolves.toMatchObject(
+      expect.objectContaining({
+        slug,
+        title: 'Lighthouse',
+        description: 'Chrome lighthouse CLI as code-pushup plugin',
+        duration: expect.any(Number),
+        date: expect.any(String),
+        audits: expect.any(Array),
+        groups: expect.any(Array),
+      }),
+    );
+  });
```

Comment:
In order to keep the E2E execution consistent, one should only interact via the CLI using `onlyPlugins` and provide a code pushup config. See ESLint E2E test [here](https://github.com/code-pushup/cli/blob/main/e2e/cli-e2e/tests/collect.e2e.test.ts#L30-L35).

Reply:
see above i removed the execution